proposals/mmmm-spawn-let.md

```diff
@@ -0,0 +1,701 @@
+  spawn let y = ...
+  // x and y execute in parallel; this is equal to the default semantics
+}
+
+actor Worker {
+  func work(first: Work, second: Work) async {
+    await withTask(executor: self.serialExecutor) {
+      // using any serial executor, will cause the tasks to be completed one-by-one,
+      // concurrently, however without any real parallelism.
+      spawn let x = process(first)
+      spawn let y = process(second)
+      // x and y do NOT execute in parallel
+    }
+  }
+}
+```
+
+The details of the API remain to be seen, but the general ability to specify an executor for child-tasks is useful and will be considered in the future.
+
+## Alternatives considered
+
+### Alternative name: `async let`
+
+The feature was previously known as `async let` yet as the work on Swift concurrency continued, based on community feedback as well as our own findings with regards of implied meanings of specific words used throughout all of Swift's concurrency APIs, we found that the word `async` is not quite right for this feature.
+
+As `spawn let` should be thought of a specialization of TaskGroups and the child-tasks spawned by either of them inherit the same kind of semantics from their enclosing parent task, it makes sense to share the same name across the two features.
+
+We feel that using the word `spawn` in _every_ case that involves the creation of a child-task is simple to understand and allows developers to learn and assume about both features in tandem. 
+
+### Explicit futures
+
+As discussed in the [structured concurrency proposal](nnnn-structured-concurrency.md#Prominent-futures), we choose not to expose futures or `Task.Handle`s for child tasks in task groups, because doing so either can undermine the hierarchy of tasks, by escaping from their parent task group and being awaited on indefinitely later, or would result in there being two kinds of future, one of which dynamically asserts that it's only used within the task's scope. `spawn let` allows for future-like data flow from child tasks to parent, without the need for general-purpose futures to be exposed.
+
+### "Don't spawn tasks when in cancelled parent"
+
+It would be very confusing to have automatically "not run" if the parent task were cancelled. Such semantics are offered by task groups via the `group.spawnUnlessCancelled` API, however would be quite difficult to express using plain `let` declarations, as effectively all such declarations would have to become implicitly throwing, which would sacrifice their general usability. We are convienced that following through with the co-operative cancellation strategy works well for `spawn let` tasks, because it composes well with what all asynchronous functions should be handling cancellation to begin with: only when they want to, in apropriate places within their execution, and deciding by themselfes if they prefer to throw a `Task.CancellationError` or rather return a partial result when cancellation ocurrs.
```

Comment:
`convienced` → `convinced`
`apropriate` → `appropriate`
`themselfes` → `themselves`
`ocurrs` → `occurs`

```suggestion
It would be very confusing to have `spawn let` tasks automatically "not run" if the parent task were cancelled. Such semantics are offered by task groups via the `group.spawnUnlessCancelled` API, however would be quite difficult to express using plain `let` declarations, as effectively all such declarations would have to become implicitly throwing, which would sacrifice their general usability. We are convinced that following through with the co-operative cancellation strategy works well for `spawn let` tasks, because it composes well with how all asynchronous functions should be handling cancellation to begin with: only when they want to, in appropriate places within their execution, and deciding by themselves if they prefer to throw a `Task.CancellationError` or rather return a partial result when cancellation occurs.
```